Pelias.podspec

```diff
@@ -12,13 +12,14 @@ Pod::Spec.new do |s|
   s.requires_arc = true
   s.ios.deployment_target = '9.3'
   s.osx.deployment_target = '10.11'
+  s.pod_target_xcconfig = { 'SWIFT_VERSION' => '3.0' }
 
   # Exclude optional Convenience modules
   s.default_subspec = 'Core'
 
   ### Subspecs
 
   s.subspec 'Core' do |cs|
-    cs.source_files   = 'src/*.swift'
```

Comment:
Nice catch, I didn't realize it required a specific directory. It seems like either `src` or `Sources` is acceptable but I chose `Sources` because its documented: https://swift.org/package-manager/#example-usage

I updated it to be `Sources/Core` after getting this error running `swift build`:
<img width="1165" alt="screen shot 2017-04-26 at 10 25 55 am" src="https://cloud.githubusercontent.com/assets/494323/25445840/c39c161e-2a6c-11e7-8270-e53c426c8e76.png">